ci/citest/acceptance-test-esxi/tests/01_start_and_terminate_esxi_instance_test.go

```diff
@@ -0,0 +1,24 @@
+// +build acceptance
+
+package tests
+
+import (
+	"strings"
+	"testing"
+	"time"
+
+	tests "github.com/axsh/openvdc/ci/citest/acceptance-test/tests"
+)
+
+func TestEsxiInstance(t *testing.T) {
+	stdout, _ := tests.RunCmdAndReportFail(t, "openvdc", "run", "centos/7/esxi", `{"interfaces":[{"Type":"veth","Ipv4Addr":"192.168.2.123"}]}`)
+	instance_id := strings.TrimSpace(stdout.String())
+
+	_, _ = tests.RunCmdAndReportFail(t, "openvdc", "show", instance_id)
+	tests.WaitInstance(t, 5*time.Minute, instance_id, "RUNNING", []string{"QUEUED", "STARTING"})
+
+	_, _ = tests.RunCmdWithTimeoutAndReportFail(t, 10, 5, "openvdc", "destroy", instance_id)
+	tests.WaitInstance(t, 5*time.Minute, instance_id, "TERMINATED", nil)
+
+	//TODO: Use Esxi api to make sure that the vm gets properly created and destroyed.
```

Comment:
Why not do this before merge? I'm afraid this will be forgotten about and might come back to bite us otherwise.

Reply:
As for the ip wait command: This used to be necessary, but indeed not anymore now after a few changes. I'll remove it.